data/SAbDab_Chen/meta.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: SAbDab_Chen
+description: "Antibody data from Chen et al, where they process from the SAbDab. \n        From an initial dataset of 3816 antibodies, they retained 2426\
+    \ antibodies\n        that satisfy the following criteria: 1. \n        have both sequence (FASTA) and Protein Data Bank (PDB) structure files,\n  \
+    \      2. contain both a heavy chain and a light chain, and 3. \n        have crystal structures with resolution < 3 Å. \n        The DI label is derived\
+    \ from BIOVIA's pipelines."
+targets:
+    - id: developability
+      description: functional antibody candidate to be developed into a manufacturable(1), or not(0)
+      units: ''
+      type: categorical
+      names:
+          - antibody developability
+          - monoclonal anitbody
+          - functional antibody candidate
+          - manufacturable, stable, safe, and effective antibody drug
+      uris:
+          - https://rb.gy/idkdqp
+          - https://rb.gy/b8cx8i
+identifiers:
+    - id: antibody_pdb_ID
+      type: Other
```

Comment:
are those IDs chemically meaningful or just some identifier number?

Reply:
yeah, the pdb id

Reply:
So should we keep them or remove them?